doc/source/getting_started/user_guide.rst

```diff
@@ -1,33 +1,30 @@
-===============
-Getting started
-===============
+==========
+User guide
+==========
 
-To run the Antenna Toolkit, you must have a licensed copy of AEDT installed.
 You have multiple options for installing and launching this toolkit:
 
-- You can install the toolkit directly in AEDT via an installation script and then launch it
-  as a wizard. For more information, see :ref:`install-toolkit-AEDT`.
-- You can install the toolkit from the AEDT console and then launch it as a wizard. For more
-  information, see :ref:`install_toolkit_console_ui`.
-- You can install and launch the toolkit directly from a Python console and then use the toolkit's APIs.
+- You can install the toolkit directly in AEDT via an installation script and then launch it as a wizard.
+  For more information, see :ref:`install-toolkit-AEDT`.
+- You can install the toolkit from a Python console and then launch Antenna wizard.
+  For more information, see :ref:`install_toolkit_console_ui`.
+- You can install the toolkit from a Python console and then use the toolkit's APIs.
   For more information, see :ref:`install_toolkit_console_api`.
 
 .. _install-toolkit-AEDT:
 
-How to install directly in AEDT and launch as a wizard
-------------------------------------------------------
+Install toolkit in AEDT and launch the Antenna wizard
+-----------------------------------------------------
 
 You can install the Antenna Toolkit directly in AEDT using the base
```

Comment:
```suggestion
You can install the AEDT Antenna Toolkit directly in AEDT using the base
```
We have some naming issues that should be cleared up in the doc in a separate PR. Based on the repo name and the title on the doc landing page, the name of this product is AEDT Antenna Toolkit. It should always be written and capitalized like that. Once this PR is merged, ping me to do an overall documentation review.